odd-platform-api/src/main/java/com/provectus/oddplatform/repository/DatasetFieldRepositoryImpl.java

```diff
@@ -166,4 +176,85 @@ public void updateSearchVectors(final long datasetFieldId) {
             Map.of(labelName, LABEL.NAME)
         ).execute();
     }
+
+    @Override
+    @Transactional
+    public DatasetFieldDto updateDatasetField(final long datasetFieldId,
+                                              final DatasetFieldUpdateFormData datasetFieldUpdateFormData) {
+        setDescription(datasetFieldId, datasetFieldUpdateFormData.getDescription());
```

Comment:
If we want to update only labels description will be null in this case. Should we perform update in this case?